### PR TITLE
V0.4 Complete Command

### PR DESCRIPTION
**Pull request by akd13**
More tests to be added later if possible
Part of #121

### CODE REVIEW

File: src/main/java/seedu/multitasky/logic/commands/CompleteByFindCommand.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+import seedu.multitasky.commons.core.Messages;
+import seedu.multitasky.logic.commands.exceptions.CommandException;
+import seedu.multitasky.logic.parser.CliSyntax;
+import seedu.multitasky.model.entry.Entry;
+import seedu.multitasky.model.entry.ReadOnlyEntry;
+import seedu.multitasky.model.entry.exceptions.DuplicateEntryException;
+import seedu.multitasky.model.entry.exceptions.EntryNotFoundException;
+
+// @@author A0140633R - reused
```
Comment (akd13) [inline src/main/java/seedu/multitasky/logic/commands/CompleteByFindCommand.java]: Is this alright? @kevinLamKB

Comment (mattheuslee) [inline src/main/java/seedu/multitasky/logic/commands/CompleteByFindCommand.java]: don't put whitespace after your matric number, the collator will stop there.

File: src/main/java/seedu/multitasky/logic/commands/CompleteByIndexCommand.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import seedu.multitasky.commons.core.index.Index;
+import seedu.multitasky.logic.commands.exceptions.CommandException;
+import seedu.multitasky.logic.parser.ParserUtil;
+import seedu.multitasky.logic.parser.Prefix;
+import seedu.multitasky.model.entry.Entry;
+import seedu.multitasky.model.entry.ReadOnlyEntry;
+import seedu.multitasky.model.entry.exceptions.DuplicateEntryException;
+import seedu.multitasky.model.entry.exceptions.EntryNotFoundException;
+
+// @@author A0140633R - reused
```
Comment (mattheuslee) [inline src/main/java/seedu/multitasky/logic/commands/CompleteByIndexCommand.java]: same as above

File: src/main/java/seedu/multitasky/logic/commands/CompleteByFindCommand.java (hunk 1 above)
Comment (mattheuslee) [reply src/main/java/seedu/multitasky/logic/commands/CompleteByFindCommand.java]: if you're reusing his code put your own matric number followed by reused.

File: src/main/java/seedu/multitasky/logic/parser/CompleteCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        return argMultimap;
+    }
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the CompleteCommand and returns a
+     * CompleteCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    // @@author A0140633R - reused
```
Comment (mattheuslee) [inline src/main/java/seedu/multitasky/logic/parser/CompleteCommandParser.java]: same as above

**Comment by kevinLamKB** [comment]
actually why not just use ur matric number @akd13 since technically you'll be the one in charge of updating and correcting these features anyway

**Comment by kevinLamKB** [comment]
ah actually i think you should follow ping chan's format , use ur matric number but put a -reused behind

**Comment by mattheuslee** [comment]
@kevinLamKB @akd13 yup that's how you're supposed to do it. See [V0.4 collation tags](http://www.comp.nus.edu.sg/~cs2103/AY1617S4/contents/handbook.html#handbook-project-v04)

**Review by mattheuslee — APPROVED** [review]

**Review by ChuaPingChan — APPROVED** [review]

**Review by kevinLamKB — APPROVED** [review]

**Review by kevinLamKB — APPROVED** [review]

**Pull request merged**